NEWS.md

```diff
@@ -9,6 +9,16 @@
 
 ## New Features
 
+- `o.format_date` as an alternative to `o.format`
+  The actual behavior of `format` is inconsistent across DB vendors: in mysql we
+  can format dates and numbers with it, and in the other ones we only format
+  dates.
+  
+  We're planning on normalizing this behavior. We want `format` to "cleverly"
+  format dates or numbers, and `format_date` / `format_number` to strictly
+  format dates / numbers.
+  
+  The introduction of `format_date` is the first step in this direction.
```

Comment:
This is not right.

I believe we already had a release with `o.present` etc., so we should have a new `# Release` section, and the previous one should be labeled with the name of the release.

Reply:
I rewrote it. Could you please take a look at it?

Reply:
Fine, thanks!